intx_sdk/view_max_loan_availability.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2025-present Coinbase Global, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from dataclasses import dataclass
+from typing import List, Optional
+from intx_sdk.base_response import BaseResponse
+from intx_sdk.client import Client
+from intx_sdk.credentials import Credentials
+
+
+@dataclass
+class ViewMaxLoanAvailabilityRequest:
```

Comment:
this one the naming is also rough. max adds no value or meaning. portfolio and asset are required, but not mentioned. ts I named it getAssetLoanAvailability (I had the portfolioService for the namespace)